doc/release/1.3.0-notes.rst

```diff
@@ -24,41 +24,575 @@ For running on PyPy, PyPy3 6.0+ and NumPy 1.15.0 are required.
+been deprecated since v0.19.0 and the latter has been deprecated since v1.0.0.
+Similarly, aliases from ``scipy.misc`` (``comb``, ``factorial``,
+``factorial2``, ``factorialk``, ``logsumexp``, ``pade``, ``info``, ``source``,
+``who``) which have been deprecated since v1.0.0 are removed. 
+`SciPy documentation for
 v1.1.0 <https://docs.scipy.org/doc/scipy-1.1.0/reference/misc.html>`__
 can be used to track the new import locations for the relocated functions.
 
+`scipy.linalg` changes
+----------------------
+
+For ``pinv``, ``pinv2``, and ``pinvh``, the default cutoff values are changed 
+for consistency (see the docs for the actual values).
+
+`scipy.stats` changes
+---------------------
+
+Previously, ``ks_2samp(data1, data2)`` would run a two-sided test and return 
+the approximated p-value. The new signature, ``ks_2samp(data1, data2,
+alternative="two-sided", method="auto"``, still runs the two-sided tests but 
+returns the exact p-value for small samples and the approximated value for 
+large samples. ``method = "asymp"`` would be equivalent to the old version but 
+``auto`` is the better choice.
 
 Other changes
 =============
 
+Our tutorial has been expanded with a new section on global optimizers
+
+There has been a rework of the ``stats.distributions`` tutorials.
+
+`scipy.optimize` now correctly sets the convergence flag of the result to 
+``CONVERR``, a convergence error, for bounded scalar-function root-finders 
+if the maximum iterations has been exceeded, ``disp`` is false, and 
+``full_output`` is true.
+
+`scipy.optimize.curve_fit` no longer fails if ``xdata`` and ``ydata`` dtypes 
+differ; they are both now automatically cast to ``float64``.
+
+`scipy.ndimage` functions including ``binary_erosion``, ``binary_closing``, and 
+``binary_dilation`` now require an integer value for the number of iterations,
+which alleviates a number of reported issues
+
+Fixed normal approximation in case ``zero_method == "pratt"`` in 
+`scipy.stats.wilcoxon`
+
+Fixes for incorrect probabilities, broadcasting issues and thread-safety 
+related to stats distributions setting member variables inside ``_argcheck()``
+
+`scipy.optimize.newton` now correctly raises a ``RuntimeError``, when default 
+arguments are used, in the case that a derivative of value zero is obtained,
+which is a special case of failing to converge
+
 
 Authors
 =======
 
+* ananyashreyjain +
+* ApamNapat +
+* Scott Calabrese Barton +
+* Christoph Baumgarten
+* Peter Bell +
+* Jacob Blomgren +
+* Doctor Bob +
+* Mana Borwornpadungkitti +
+* Matthew Brett
+* Evgeni Burovski
+* CJ Carey
+* Vega Theil Carstensen +
+* Robert Cimrman
+* Forrest Collman +
+* Pietro Cottone +
+* David +
+* Idan David +
+* Christoph Deil
+* Dieter Werthmüller
+* Conner DiPaolo +
+* Dowon
+* Michael Dunphy +
+* Peter Andreas Entschev +
+* Gökçen Eraslan +
+* Johann Faouzi +
+* Yu Feng
+* Piotr Figiel +
+* Matthew Flamm +
+* Franz Forstmayr +
+* GitHub
```

Comment:
can we get rid of this one? must be a strange commit somewhere that needs a .mailmap catch or something

Reply:
This one I had to remove manually for now--I've also revised with another mailmap update to deal with 1 person attributed 3 times in different ways.